docs/advanced_usage.md

```diff
@@ -24,3 +26,31 @@ relayed_packets{chain="cosmoshub-4",channel="channel-141",path="hubosmo",port="t
 relayed_packets{chain="osmosis-1",channel="channel-0",path="hubosmo",port="transfer",type="acknowledge_packet"} 36
 relayed_packets{chain="osmosis-1",channel="channel-0",path="hubosmo",port="transfer",type="recv_packet"} 35
 ```
+
+---
+
+## Auto Update Light Client
+
+By default, the Relayer will automatically update clients (`msgUpdateClient`) if the client has <= 1/3 of its trusting period left. 
+
+> NOTE: The trusting period of the corresponding client is restored with each transaction the Relayer relays. In other words, every time the Relayer relays a message, it also sends a `msgUpdateClient` message restarting the time to the clients expiration. 
```

Comment:
This almost makes it sound like the client trusting period is only being updated by one instance of the relayer. Rewording this to something like "... restored with each transaction any active relayer relays." may be more concise with what's going on.

Also small nit but `MsgUpdateClient` is the spelling found in ibc-go

Reply:
Good call here Justin.
I added an "*" sentence to the bottom of this doc to address technicalities.